_config.yml

```diff
@@ -0,0 +1,30 @@
+title: Meteor Demo
+author: AIOps team
+# logo: logo.png
+execute:
+  execute_notebooks: false
+
+# Only build files specified by the _toc
+only_build_toc_files: true
+
+# Exclude any additional file types being built
+exclude_patterns: [.github/*, manifests/*,  CHANGELOG.md, docs/images/*]
+
+# Information about where the book exists on the web
+repository:
+  url: https://github.com/aicoe-aiops/meteor  # Online location of your book
+  path_to_book: ""  # Optional path to your book, relative to the repository root
+  branch: master  # Which branch of the repository should be used when creating links (optional)
+
+# Add GitHub buttons to your book
+# See https://jupyterbook.org/customize/config.html#add-a-link-to-your-repository
+html:
+  use_issues_button: true
+  use_repository_button: true
+  google_analytics_id: ""  # A GA id that can be used to track book views.
+
+# Launch button settings
+launch_buttons:
+  notebook_interface        : jupyterlab  # The interface interactive links will activate ["classic", "jupyterlab"]
+  jupyterhub_url            : https://jupyterhub-opf-jupyterhub.apps.zero.massopen.cloud  # The URL of the JupyterHub (e.g., https://datahub.berkeley.edu)
```

Comment:
This needs to be updated to reflect the smaug cluster. Do we know if `https://jupyterhub-opf-jupyterhub.apps.smaug.massopen.cloud` will be the name? Can we inject env variables here somehow to maintain this repo without updating the hand coded the jupyter_url if we want to switch clusters?